Project3/styles.css

```diff
@@ -0,0 +1,34 @@
+/*.img-timeline 
+    background-image: url("TimelineArrow-01.png");
+    background-repeat: repeat-y;
+    position: absolute;
+	width: 30px;
+    display: inline;
+*/
+body {
+    background-image: url("TimelineArrow-01.png");
+    background-repeat: repeat-y;
+    position: relative;
+	width: 30px;
+    display: inline;
+}
+.card {
+    min-width: 200px;
+    max-width: 800px;
```

Comment:
If you include `min-width` and `max-width`, then generally `max-width` wins and `min-width` becomes meaningless. So, here you can either:
1. Just write `width: 800px;`
1. Write a relative width (like `width: 60%;`) and then use min/max to make sure it doesn't go overboard.